app/views/items/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="wrapper">
+  <div class="row">
+    <div class="col-sm-6 col-md-4">
+      <div class="thumbnail">
+        <img src=<%= @item.image_url %> alt=<%= @item.title %>>
+      </div>
+    </div>
+    <div class="col-sm-6 col-md-8">
+      <p><%= link_to @item.category_name, category_path(@item.category) %></p>
+      <h2><%= @item.title %></h2>
+      <p><%= @item.description %></p>
+      <div>
+        <h4><%= number_to_currency(@item.price_in_dollars) %></h4>
+        <% if @item.retired == false %>
```

Comment:
this is too much logic in your view. i suggest creating a method on `Item` called `retired?` that does this for you 😄